### DISCUSSION TITLE
Multiple Templates for a single Content Type

### DISCUSSION THREAD

**Discussion by Jeremy-Code-F** [OrchardCMS/OrchardCore#10021] [Q&A] [answered]
Hi, I'm trying to understand how we can utilize multiple view templates for a single content type

e.g. the default template for a content type x should be y but how can I say on a specific area on the site it should use z view template.

I see there are 'Alternates' that seem like would fit what I'm trying to accomplish 

> Orchard Core can match with many templates for the same Shape.
These potential templates are called Alternates.

but I'm not sure I see how to tell Orchard, for this render of this content type use this alternate.

Thanks in advance!

**Comment by ns8482e** [comment, marked as answer]
Use [Placements ](https://docs.orchardcore.net/en/latest/docs/reference/core/Placement/#format)

Using placement you can define different `alternates` or `shape` based on different criteria like `path`, `displayType` , `contentType`, `differentiator`

**Reply by Jeremy-Code-F** [reply]
Thanks! I will do some looking into Placements and see if I that would work for my use case...do you happen to know if there is any through the web way of changing what template is used? Similar to DNN's visualizers? 
https://www.dnnsoftware.com/cms-features/visualizers

**Reply by ns8482e** [reply]
Orchard Core doesn't have designer like that however, there are different displayType  available e.g. `Summary`,  `Detail`.  So if you have `ContentItem` named `faq`  and for `summary` display type you can create  shape  template `faq_summary` or liquid file `faq.summary.liquid`

Using `Templates` feature you can manage your templates in Admin https://docs.orchardcore.net/en/latest/docs/reference/modules/Templates/
